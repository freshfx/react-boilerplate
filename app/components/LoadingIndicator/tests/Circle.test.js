import React from 'react'
import {render} from '@testing-library/react'

import Circle from '../Circle'

describe('Circle', () => {
  it('should match default snapshot', () => {
    const {container} = render(<Circle />)
    expect(container.firstChild).toMatchSnapshot()
  })

  it('should match snapshot with rotate', () => {
    const {container} = render(<Circle rotate={10} />)
    expect(container.firstChild).toMatchSnapshot()
  })

  it('should match snapshot with rotate & delay', () => {
    const {container} = render(<Circle rotate={10} delay={100} />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
