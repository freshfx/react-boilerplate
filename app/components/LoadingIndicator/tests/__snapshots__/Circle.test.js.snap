// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle should match default snapshot 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  position: absolute;
  left: 0;
  top: 0;
}

0 .c0:before {
  content: '';
  display: block;
  margin: 0 auto;
  width: 15%;
  height: 15%;
  background-color: #999;
  border-radius: 100%;
  -webkit-animation: evVlQQ 1.2s infinite ease-in-out both;
  animation: evVlQQ 1.2s infinite ease-in-out both;
}

<div
  class="c0"
/>
`;

exports[`Circle should match snapshot with rotate & delay 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  position: absolute;
  left: 0;
  top: 0;
  -webkit-transform: rotate(10deg);
  -ms-transform: rotate(10deg);
  -webkit-transform: rotate(10deg);
  -ms-transform: rotate(10deg);
  transform: rotate(10deg);
}

.c0:before {
  content: '';
  display: block;
  margin: 0 auto;
  width: 15%;
  height: 15%;
  background-color: #999;
  border-radius: 100%;
  -webkit-animation: evVlQQ 1.2s infinite ease-in-out both;
  animation: evVlQQ 1.2s infinite ease-in-out both;
  -webkit-animation-delay: 100s;
  -webkit-animation-delay: 100s;
  animation-delay: 100s;
}

<div
  class="c0"
/>
`;

exports[`Circle should match snapshot with rotate 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  position: absolute;
  left: 0;
  top: 0;
  -webkit-transform: rotate(10deg);
  -ms-transform: rotate(10deg);
  -webkit-transform: rotate(10deg);
  -ms-transform: rotate(10deg);
  transform: rotate(10deg);
}

.c0:before {
  content: '';
  display: block;
  margin: 0 auto;
  width: 15%;
  height: 15%;
  background-color: #999;
  border-radius: 100%;
  -webkit-animation: evVlQQ 1.2s infinite ease-in-out both;
  animation: evVlQQ 1.2s infinite ease-in-out both;
}

<div
  class="c0"
/>
`;
